modules/Login/src/server/routes/loginRoutes.coffee

```diff
@@ -90,20 +90,20 @@ exports.changePost = (req, res) ->
 	res.redirect '/passwordChange'
 
 exports.logout = (req, res) ->
-	req.logout()
-	if config.all.server.security.saml.use == true && config.all.server.security.saml.logoutRedirectURL?
-		redirectMatch = config.all.server.security.saml.logoutRedirectURL
-	else 
-		redirectMatch = req.originalUrl.match(/^\/logout\/(.*)\/?$/i)
-		if redirectMatch?
-			redirectMatch = redirectMatch[1]
-		else
-			if config.all.client.basePath?
-				redirectMatch = config.all.client.basePath
+	req.logout =>
+		if config.all.server.security.saml.use == true && config.all.server.security.saml.logoutRedirectURL?
```

Comment:
Diff is a little funky but all I did was pass logout a function with `=>` and tab over the rest of the code here.